milli/src/search/new/graph_based_ranking_rule.rs

```diff
@@ -162,12 +164,16 @@ impl<'ctx, G: RankingRuleGraphTrait> RankingRule<'ctx, QueryGraph> for GraphBase
         // Then pre-compute the cost of all paths from each node to the end node
         let all_costs = graph.find_all_costs_to_end();
 
+        let next_max_cost =
+            all_costs.get(graph.query_graph.root_node).iter().copied().max().unwrap_or(0) + 1;
+
```

Comment:
why "next" max cost?

Good that it is computed at `start_iteration` 👍

Reply:
Because of the `+1`. It is computed as the max cost of all the cost, + 1.

Initially I just called that `max_cost`, but it felt misleading as it is not one of the costs.

Feel free to suggest something clearer, or maybe I should add some comment here?

Maybe `ceiling_cost` but uhhh